### DISCUSSION TITLE
Welcome to jmeter-grpc-request Discussions!

### DISCUSSION THREAD

**Discussion by anhldbk** [zalopay-oss/jmeter-grpc-request#22] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by steverawlins-zebra** [comment]
I've been using this plug-in for over a year, load-testing services for my employer.
